content/en/docs/guides/build-deploy-software-via-helm-charts-with-ocm-101.md

```diff
@@ -0,0 +1,629 @@
+---
+title: "Build & Deploy Infrastructure via Helm Charts with OCM" 
+description: ""
+lead: ""
+date: 2024-03-19T10:36:48+01:00
+lastmod: 2024-03-19T10:36:48+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "build-deploy-software-via-helm-charts-with-ocm"
+weight: 101
+toc: true
+---
+
+## Introduction
+
+Let's illustrate a very simple "Hello World" example application and show how to leverage OCM to build and deploy a Helm Chart and OCI Image on a local `kind` k8s cluster.
+The topic of `ocm` [`localization`](https://ocm.software/docs/guides/complex-component-structure-deployment/#localization) and [`configuration`](https://ocm.software/docs/guides/complex-component-structure-deployment/#configuration) is NOT part of this very simple example and is part of other in deepth guides.
+
+As base we use the `podinfo` application.
+All files can be found [here](https://github.com/stb1337/ocm-hello-world-v1).
```

Comment:
I propose to re-activate the https://github.com/open-component-model/ocm-examples repository and move all files required for examples linked on the website to it. The podinfo component file(s) may end up [here](https://github.com/open-component-model/ocm-examples/tree/main/components) and all other required files under the kubernetes folder.

Reply:
I can move everything there, but therefore this repository needs to be public again. 
Additionally, I would add everything into `[ocm-examples](https://github.com/open-component-model/ocm-examples/tree/main)/[components](https://github.com/open-component-model/ocm-examples/tree/main/components)/guide-walkthrough-helm-chart` and `[ocm-examples](https://github.com/open-component-model/ocm-examples/tree/main)/[kubernetes](https://github.com/open-component-model/ocm-examples/tree/main/kubernetes)/guide-walkthrough-helm-chart` ?

Reply:
done, I have moved all files to ocm-example -> https://github.com/open-component-model/ocm-examples/pull/25

Reply:
@morri-son updated references to ocm-example repository. Please resolve conversation and approve PR.